etn-ccis/blui-icons
# BLUI-4717: added 9 new icons

Fixes BLUI-4717, fixes #308 



## Changes proposed in this Pull Request:

- Added 9 new icons: Ethernet Port, Skillet, Skillet Cooktop, Cooking, Stockpot, Stove, Wind Turbine, Water Energy, IOT Scene.
- Replaced Tom with Ekta in the code owners.

## Screenshots / Screen Recording (if applicable)

![image](https://github.com/etn-ccis/blui-icons/assets/8997218/0e2c3aa6-7815-41b6-b38a-26ecd31aefce)




## To Test:

- Verify that all the icon files have been correctly generated for each package (SVG, mui, PNG, rn-vector, icon font)
- `yarn && yarn start`
- **fixed now** ~Normally you would do yarn start. But for some reason `yarn start` does not work on my machine (#308)~
